R/translate-sql-window.R

```diff
@@ -253,19 +252,15 @@ local_context <- function(x, env = parent.frame()) {
 # Where translation -------------------------------------------------------
 
 uses_window_fun <- function(x, con) {
-  if (is.null(x)) return(FALSE)
```

Comment:
`uses_window_fun()` is only called by `sql_build.op_filter()` and there it is always passed quosures.